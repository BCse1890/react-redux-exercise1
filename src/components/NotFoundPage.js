import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
    <div>
        404! Page not found.<Link to="/">Home page </Link>
    </div>
);

export default NotFoundPage;