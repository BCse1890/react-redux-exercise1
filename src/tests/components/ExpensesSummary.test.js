import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";
import { connect } from "tls";

test("should correctly render expenses summary with single expense", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={1} expensesTotal={235} />
  );
  expect(wrapper).toMatchSnapshot();
});

test("should correctly render expenses summary with multiple expenses", () => {
  const wrapper = shallow(
    <ExpensesSummary expenseCount={18} expensesTotal={43829323892} />
  );
  expect(wrapper).toMatchSnapshot();
});
