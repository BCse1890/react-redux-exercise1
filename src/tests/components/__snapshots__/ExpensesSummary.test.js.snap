// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    You have 
    18
     
    expenses
     with total value:
    $438,293,238.92
  </h1>
</div>
`;

exports[`should correctly render expenses summary with single expense 1`] = `
<div>
  <h1>
    You have 
    1
     
    expense
     with total value:
    $2.35
  </h1>
</div>
`;
